Formula/delve.rb

```diff
@@ -0,0 +1,17 @@
+class Delve < Formula
+  desc "Debugger for the Go programming language"
+  homepage "https://github.com/go-delve/delve"
+  url "https://github.com/go-delve/delve/archive/v1.6.0.tar.gz"
+  sha256 "a10a6fc40d87572c6d3f3becdb1a289269e17526d038749f2fa04dd9f591f26a"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "-o", bin/"dlv", "./cmd/dlv"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/dlv version")
```

Comment:
Is this a match or is it identical? If a match: a more specific match would be preferable (so 1.6.0 doesn't match 1.6.0.1) if not a full string equality comparison, ideally.

Reply:
OK, I made a more specific version match. Please review.